subcommands/targets/show.go

```diff
@@ -243,6 +258,25 @@ func doShowSboms(cmd *cobra.Command, args []string) {
 	downloadPath, _ := cmd.Flags().GetString("download")
 
 	_, _, targets := getTargets(factory, prodTag, version)
+	if len(args) == 3 {
+		path := fmt.Sprintf("%s/%s", args[1], args[2])
+		for name := range targets {
+			packages, err := api.TargetSbomPackages(factory, name, path)
+			subcommands.DieNotNil(err)
+			t := tabby.New()
+			t.AddHeader("NAME", "VERSION", "LICENSE")
+			for _, pkg := range packages {
+				t.AddLine(pkg.Name, pkg.VersionInfo, pkg.License())
+			}
+			t.Print()
+			// all targets come from the same CI build with the same SBOM return
+			return
+		}
```

Comment:
Does it make sense for a user to use the `--download` option in this code path?

- If yes - please, implement it.
- If no - please, show an error message if it was provided.